iohk/common/Types.hs

```diff
@@ -140,6 +141,7 @@ data Environment
   | Staging
   | Testnet
   | Development
+  | Poc
```

Comment:
I don't think we should add the POC to io. In my opinion, the POC is temporary and will never be merged into master.